aspnetcore/migration/mvc.md

```diff
@@ -3,18 +3,189 @@ title: Migrate from ASP.NET MVC to ASP.NET Core MVC
+
+## Migrate Controllers and views
+
+In the ASP.NET Core project a new empty controller class and view class would be added to serve as placeholders using the same names as the controller and view classes in any ASP.NET MVC project you migrate from.
+
+The ASP.NET Core *WebApp1* project already includes a minimal example controller and view by the same name as the ASP.NET MVC project. So those will serve as placeholders for the ASP.NET MVC controller and views to be migrated from the ASP.NET MVC *WebApp1* project.  
+
+1. Copy the methods from the ASP.NET MVC `HomeController` to replace the new ASP.NET Core `HomeController` methods. There's no need to change the return type of the action methods. The ASP.NET MVC built-in template's controller action method return type is [ActionResult](https://msdn.microsoft.com/library/system.web.mvc.actionresult(v=vs.118).aspx); in ASP.NET Core MVC, the action methods return `IActionResult` instead. `ActionResult` implements `IActionResult`.
+1. In the ASP.NET Core project, right-click the **Views** > **Home** folder, select  **Add** > **Existing Item...**.
+1. In the **Add Existing Item dialog**, navigate to the ASP.NET MVC WebApp1 project > Views > Home directory.
+1. Select the *About.cshtml*, *Contact.cshtml*, and *Index.cshtml* Razor view files, then select **Add**, replacing the existing files.
+1. Run the ASP.NET Core app.
+
+See [Controllers](xref:mvc/controllers/actions) and [Views](xref:mvc/views/overview) for more information.
+
+## Test each method
+
+The layout file and styles have not been migrated yet, so the rendered views only contain the content in the view files.  Also, the layout file generated links for the `About` and `Contact` views will not be available yet.
+
+* Invoke the rendered views from the browser on the running ASP.NET core app by replacing the current port number with the port number used in the ASP.NET core project. For example: `https://localhost:44375/home/about`.
+
+See [ASP.NET Core Controllers](xref:mvc/controllers/actions) and [ASP.NET Core Views](xref:mvc/views/overview) for more information.
+
+## Migrate Static content
+
+In previous versions of ASP.NET MVC, static content was hosted from the root of the web project and was intermixed with server-side files. In ASP.NET Core, static content is hosted in the **wwwroot** folder.
+
+Copy the static content from the ASP.NET MVC *WebApp1* project to the *wwwroot* folder in the ASP.NET Core *WebApp1* project:
+
+1. In the ASP.NET Core project, right-click the **wwwroot** folder, select  **Add** > **Existing Item...**.
+1. In the **Add Existing Item dialog**, navigate to the ASP.NET MVC *WebApp1* project.
+1. Select the *favicon.ico* file, then select **Add**, replacing the existing file.
+
+## Migrate the layout files
+
+Copy the ASP.NET MVC project layout files to the ASP.NET Core project:
+
+1. In the ASP.NET Core project, right-click the **Views** folder, select  **Add** > **Existing Item...**.
```

Comment:
```suggestion
1. In the ASP.NET Core project, right-click the **Views** folder, select  **Add** > **Existing Item**.
```

Reply:
@scottaddie, thanks. Folders use italics:  What about the following situation where we are technically referring to a UI element, but it is a folder?  Still go with italics?

"In the ASP.NET Core project, right-click the *wwwroot* folder, select  **Add** > **Existing Item...**.

Reply:
Yep, go with italics in this case